### PR TITLE
Improve and fix new shifting text in french translation

### PR DESCRIPTION
**Pull request by Nilvus**
I miss some text to fix (especially tone equalizer and liquify modules). I also found that separation between text on lines is better with the dot than a comma.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -19485,7 +19481,7 @@ msgstr ""
 " • Filtre guidé moyenné – une moyenne géométrique des méthodes « Non » et  "
 "« Filtre guidé »\n"
 " • EIGF – un Filtre Guidé Indépendant de l’Exposition, qui adoucit les "
-"ombres et les hautes lumières\n"
+"ombres et les hautes lumières \n"
```
[the remark this answers is not in the document]
Comment (Nilvus) [reply po/fr.po:19484]: Euh, je l'ai raté celui-là. Je ne sais même pas d'où il sort. Je corrige.

[the remark this answers is not in the document]
Comment (Nilvus) [reply po/fr.po]: Corrigé

**Comment by TurboGit** [comment]
I also think that all word for each action should start with a capital letter:

1: Taille & Opacité
image

2: Ajouter & Annuler
image

3: Dureté
image

4: Taille & Opacité
image

There may be others. And yes I do like the big dot, more visible separation.

**Comment by Nilvus** [comment]
Should be good now @TurboGit

**Review by TurboGit — APPROVED** [review]
Looks good, I think we should also rework the Liquify strings to be conformant to the others. That is having the action in bold first and the shortcuts after. I'll see if I can do that on the coming days.

**Comment by Nilvus** [comment]
> Looks good, I think we should also rework the Liquify strings to be conformant to the others. That is having the action in bold first and the shortcuts after. I'll see if I can do that on the coming days.

Yes, should be better to modify that on liquify.c file then translation, indeed. I don't use that module so I let you do that as you propose to! Thanks.

**Pull request merged**